tests/bluetooth/controller/ctrl_cte_req/testcase.yaml

```diff
@@ -0,0 +1,5 @@
+common:
+    tags: test_framework bluetooth bt_le_cte_req bt_ull_llcp
+tests:
```

Comment:
Should there be tests differentiating on the misc. configs (CONFIG_BT_CTLR_DF_CONN_CTE_REQ/RSP)?

Reply:
That was may aim. To split tests into two separate parts: `CTE_REQ` and `CTE_RSP`. There are different configuration for request and response parts. It would be far more readable  if one part of the communication is removed from unit tests of another. The better approach is to concentrate on tests e.g. of `CTE_REQ` and mock peripheral part with expected responses.